Examples/test-suite/common.mk

```diff
@@ -713,6 +713,13 @@ check-cpp: $(CPP_TEST_CASES:=.cpptest)
 
 check-cpp11: $(CPP11_TEST_CASES:=.cpptest)
 
+check-failing-test = \
+	$(MAKE) -s $1.$2 >/dev/null 2>/dev/null && echo "Failing test $t passed."
```

Comment:
Oops, you're right, it would be more hygienic to use `$1` here. `$t` happens to work too but it's a leftover before I extracted this command into its own variable.

I'll update it if there are no objections.